.circleci/config.yml

```diff
@@ -238,6 +238,25 @@ jobs:
       - save:
           filename: "Mattermost_Beta.apk"
 
+  build-android-unsigned:
+    executor: android
+    steps:
+      - checkout:
+          path: ~/mattermost-mobile
+      - npm-dependencies
+      - assets
+      - fastlane-dependencies:
+          for: android
+      - gradle-dependencies
+      - run:
+          working_directory: fastlane
+          name: Run fastlane to build unsigned android
+          no_output_timeout: 30m
+          command: bundle exec fastlane android unsigned || exit 1
```

Comment:
why do you need the `|| exit 1`? anything that doesn't return a 0 will show as failed on circle, so does the `||` operator

Reply:
will remove the `|| exit 1` as is not needed